pkg/volume/iscsi/iscsi.go

```diff
@@ -506,6 +519,26 @@ func getISCSISecretNameAndNamespace(spec *volume.Spec, defaultSecretNamespace st
 	return "", "", fmt.Errorf("Spec does not reference an ISCSI volume type")
 }
 
+// updateFromPV will read iSCSI Timeout settings from Persistent Volume object
+func (kvp iscsiTimeouts) getISCSITimeoutSettingInfo(spec *volume.Spec) iscsiTimeouts {
```

Comment:
Please rewrite to `func getISCSITimeoutSettingInfo(...) iscsiTimeouts`, there is no need for this to be method of `iscsiTimeouts`